EIPS/eip-7775.md

```diff
@@ -0,0 +1,212 @@
+---
+eip: 7775
```

Comment:
```suggestion
eip: 7524
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Done!